docroot/sites/default/sync/views.view.mirador_view.yml

```diff
@@ -0,0 +1,268 @@
+uuid: cce233e3-859d-40a3-b7ad-97c7d542b749
```

Comment:
@deepak-zyxware This view is already exported as a part of feature. Why do we have it in sync directory?